great_expectations_cloud/agent/event_handler.py

```diff
@@ -72,10 +73,13 @@ def get_event_action(self, event: Event) -> AgentAction[Any]:
         return action_class(context=self._context)
 
     def handle_event(self, event: Event, id: str) -> ActionResult:
```

Comment:
I added the duration calculation here inside of the `handle_event` method so that we don't include time the job might spend queued - since I don't think we would want to bill users for that.